app/views/layouts/_attention.html.erb

```diff
@@ -1,22 +1,12 @@
-<% alert_messages = [] %>
-<% handler_errors.each do |error| %>
-  <% alert_messages.push error.translate %>
-<% end %>
-
-<% alert_messages.push(flash[:alert]) if flash[:alert] %>
-
-<% notice_messages = [] %>
-<% notice_messages.push(flash[:notice]) if flash[:notice] %>
-
-<% alert_messages = alert_messages.collect{|msg| msg.gsub("contact support", mail_to("info@openstax.org", "contact support")).html_safe} %>
+<% errors = collect_errors %>
 
 <div class="row">
   <div class="col-xs-12">
     <noscript>
-      <%= attention_tag (t :".please_enable_javascript"), :alert %>
+      <%= attention_tag('please_enable_javascript', :alert) %>
```

Comment:
No not really..  At one point I played setting alerts using translation keys which the alert_tag would call `.t` on.  I later decided that was a bad idea, but looks like I forgot to switch this back.